packages/ra-ui-materialui/src/layout/Sidebar.tsx

```diff
@@ -29,22 +35,13 @@ export const Sidebar = (props: SidebarProps) => {
         >
             {children}
         </StyledDrawer>
-    ) : isSmall ? (
```

Comment:
This ternary returns the same result in both cases since 9c965a8d725977d2b48bf755ffe57274a4e6731b (1 year ago), so we can remove it.